frontend/src/api/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export async function register(name, email, password, confirmPassword) {
+    const response = await axios.post("http://localhost:8000/api/register", {
```

Comment:
Create an axios instance so you dont need to fill the base url
https://axios-http.com/docs/instance